websites/T/timolia/presence.ts

```diff
@@ -0,0 +1,73 @@
+const presence = new Presence({
+    clientId: "872421983554502717"
+  }),
+
+  browsingStamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "large_image",
+    details: document.location.href,
+    startTimestamp: browsingStamp
+  };
+
+  if (document.location.hostname === "www.timolia.de") {
+    if (document.location.pathname === "/") {
+        presenceData.startTimestamp = browsingStamp;
+        const player = document.querySelector("#liveplayercount").textContent;
+        presenceData.details = "Viewing home page";
+        presenceData.smallImageKey = "minecraft";
+        presenceData.smallImageText = `${player} online players`;
+    } else if (document.location.pathname.startsWith("/stats")) {
+        presenceData.startTimestamp = browsingStamp;
+        const stats = document.querySelector("#playername").textContent;
+        presenceData.details = "Viewing stats from:";
+        presenceData.state = stats;
+    } else if (document.location.pathname.includes("/games")) {
+        presenceData.startTimestamp = browsingStamp;
+        presenceData.details = "Viewing gamemodes";
+    } else if (document.location.pathname.includes("/account")) {
+        presenceData.startTimestamp = browsingStamp;
+        presenceData.details = "Viewing account settings";
+    }
+  } else if (document.location.hostname === "howto.timolia.de") {
+        presenceData.startTimestamp = browsingStamp;
+        presenceData.details = "HowTo - Reading:";
+        presenceData.state = document.querySelector("body > div.md-container > main > div > div.md-content > article > h1").textContent;
+    } else if (document.location.hostname === "shop.timolia.de") {
+        presenceData.startTimestamp = browsingStamp;
+        presenceData.details = "Shop - Viewing store page";
+        if (document.location.pathname.startsWith("/checkout")) {
+            presenceData.startTimestamp = browsingStamp;
+            presenceData.details = "Shop - Viewing shopping cart:";
+            presenceData.state = document.querySelector("body > div.sections-wrapper > section > div > div > div.jumbotron > div > div.packages > form > table > tbody > tr > td.col-md-6").textContent;
```

Comment:
What exactly would it show here?

Reply:
The article you are looking at

Reply:
![Screenshot 2021-08-10 000212](https://user-images.githubusercontent.com/69513317/128780150-c3788b92-491d-4528-a1d5-a2100e50d7af.png)